story001.txt

```diff
@@ -0,0 +1,16 @@
+Esta historial es de un clavo, "Clav", y su enemigo, el martillo "Martíl". Clav vivía in su casa - un paquete con 100 otros clavos. Fue una vida buena... pues, hasta que todo se cambió. Un día, una forma misteriosa apareció y tomó el paquete de clavos. Al inicio, Clav sentía emocionado... dijo, "Ay, ¡puedo ver todo el mundo, finalmente!"
+
+...Pero, eventualmente, la forma misteriosa puso el paquete en una mesa, y la adventura terminó. Clav no pudo ver la figura bien, pero él sabía que sus amigos (otros clavos) estuvieron tomado alejos. Clav trataba esconder, pero fue difícil hacerlo en un paquete pequeño, y él estuvo tomado también.
+
+Después de que la forma lo movió Clav a un cuenco, Clav conocí su enemigo . . . Martíl.
+
+"Ja ja ja... ¡Hola, clavochicos! ¿Están listos?" La voz de Matíl fue horrible, como él nunca podía tener el aire suficiente a hablar. Clav se temó. Los otros clavos sabían que Clav era el clavo más inteligente, por eso ellos lo fomentaban.
```

Comment:
I think `La voz de Matíl era horrible` instead of `fue horrible` sounds better. 
* `como si no tuviera suficiente aire par hablar`
* `Clav tuvo miedo`
* Maybe try to find a better word for `fomentaban`. I'm not sure what you're trying to say with that word.